contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -473,10 +471,6 @@ contract TrueCreditAgency is UpgradeableClaimable, ITrueCreditAgency {
         return Status.Eligible;
     }
 
-    /**
-     * @dev check if borrower score has been updated recently enough
-     * @return Whether block timestamp is less than last update + threshold
-     */
```

Comment:
I know this docstring change is irrelevant, but prettier insisted that I delete the docstring so that `yarn lint` passes. I hope you don't mind, given this whole function is hopefully going away soon.

Reply:
It's fine, just don't understand what's prettier's problem with that